### ISSUE TITLE
./configure --libdir value not passed to libite

### ISSUE THREAD

**Issue by vedranmiletic** [troglobit/inadyn#127]
It would be neat if ./configure would also pass --libdir to libite. Makefile for libite is already there in the tarball, so this might be tricky.

```
$ ./configure --enable-openssl --libdir=/usr/local/lib64
(...)
$ grep libdir Makefile libite/Makefile
(...)
Makefile:libdir = /usr/local/lib64
libite/Makefile:libdir     ?= $(prefix)/lib
(...)
```

**Comment by troglobit** [comment]
What version of Inadyn are you trying to build?  The v2.0 release will be out soon and the build system for both Inadyn and libite have changed considerably.

**Comment by vedranmiletic** [comment]
1.99.15. I can check git snapshot.

**Comment by troglobit** [comment]
Great, you need to build libite separately now ... inadyn will check for it using pkg-config.  Hope this works better for you!

Also, new .conf file format.

**Comment by troglobit** [comment]
Hi @vedranmiletic, I went ahead and installed Fedora 23 this morning. Just to check; "How hard could it possibly be?!" ... man, had I underestimated things or what?! LOL

I'm so used to Debian/Ubuntu polishing off the rough edges of things I had forgotten the perils of working on other Linux distros.  Here are a few pointers to get the GIT version of Inadyn working:

- Install [libite v1.8.0](https://github.com/troglobit/libite/releases/download/v1.8.0/libite-1.8.0.tar.xz) from tarball: `./configure && make && sudo make install`
- Install [libConfuse v3.0](https://github.com/martinh/libconfuse/releases/download/v3.0/confuse-3.0.tar.xz) from tarball: `./configure && make && sudo make install`
- Run `ldconfig` to update the .so cache ... FAIL! :disappointed: 
- Edit ldconfig's .cfg file(s): `sudo vi /etc/ld.so.conf.d/dyninst-x86_64.conf`, add
  the line `/usr/local/lib`, which is the GNU standard location for locally built package
- Run `ldconfig -v |grep libite` and get a hit this time ... libconfuse should also be found similarliy
- Clone Inadyn, to get a friendly configure script run: `./autogen.sh`
- Install `openssl-devel`: `sudo dnf install openssl-devel`
- Now, to work around yet another Fedora specific thing, run the Inadyn configure script like this: `PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:$PKG_CONFIG_PATH ./configure --enable-openssl`

May the force be with you!
